torch/ao/quantization/pt2e/utils.py

```diff
@@ -310,6 +310,16 @@ def get_aten_graph_module(
     )
     aten_pattern.graph.eliminate_dead_code()
     aten_pattern.recompile()
+
+    # ep.module() adds copy_ nodes for the mutated inputs.
+    # For patterns, it doesn't matter
```

Comment:
Can you expand on this comment, like why we need to do this in the first place, what breaks if we don't etc.